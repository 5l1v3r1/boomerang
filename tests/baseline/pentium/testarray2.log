got filename testarray2.c for main
Unknown library function proc1
Unknown library function proc2
Unknown library function proc3
after removing matching assigns (%pc).
in proc main adding addrExp r28{0} - 4 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 60 to local table
in proc main adding addrExp r28{0} - 64 to local table
in proc main adding addrExp r28{0} - 60 to local table
in proc main adding addrExp r28{0} - 64 to local table
in proc main adding addrExp r28{0} - 16 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 16 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 16 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 56 to local table
in proc main adding addrExp r28{0} - 60 to local table
in proc main adding addrExp r28{0} - 64 to local table
could not set type for unknown parameter expression m[r28{0} + 4]
could not set type for unknown parameter expression m[r28{0} + 8]
could not set type for unknown parameter expression m[r28{0} + 12]
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 16 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 8 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 16 to local table
in proc main adding addrExp r28{0} - 12 to local table
in proc main adding addrExp r28{0} - 16 to local table
in proc main adding addrExp r28{0} - 8 to local table
local edx has type int that doesn't agree with type of define void of a library, why?
local esp has type void * that doesn't agree with type of define void of a library, why?
local eax has type __size32 that doesn't agree with type of define /*signed?*/int of a library, why?
local edx has type void that doesn't agree with type of define int of a library, why?
local eax has type /*signed?*/int that doesn't agree with type of define __size32 of a library, why?
local edx has type int that doesn't agree with type of define __size32 of a library, why?
