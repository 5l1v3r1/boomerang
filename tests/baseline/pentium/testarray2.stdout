Boomerang alpha 0.3.1 09/Sep/2006
loading...
decoding entry point...
decoding anything undecoded...
finishing decode...
found 3 procs
decompiling...
 considering main
  considering mid
  decompiling mid
  considering fst
  decompiling fst
 decompiling main
global type analysis for main
global type analysis for mid
global type analysis for fst
generating code...
output written to ./tests/outputs/pentium/testarray2
completed in 0 secs.
